### ISSUE TITLE
background=light not properly set for nvim running within nvim

### ISSUE THREAD

**Issue by dstein64** [neovim/neovim#15365]
### Neovim Version

NVIM v0.5.0

### Operating system/version:

Ubuntu 21.04

### Terminal name/version:

Konsole 20.12.3

### TERM environment variable

xterm-256color

### Installation

appimage

### Steps to reproduce

First change the terminal's color scheme to use a light background.

Then:

```vim
nvim -u NORC
" Confirm background=light
:set background?
" Run Neovim within Neovim
:term nvim
" Focus inner Neovim (alternatively could press "i")
:startinsert
" Check background
:set background?
```

### Vim

The expected behavior is encountered when running 1) Vim within Vim, 2) Neovim within Vim, or 3) Vim within Neovim.

### Expected behavior

`background=light` within the inner Neovim.

### Actual behavior

`background=dark` within the inner Neovim.

**Comment by justinmk** [comment]
`:terminal` is likely not responding to the bg color request from the nested `nvim`. https://github.com/neovim/neovim/blob/5a111c1b02bbfbc2b42df11d7205153be0893dff/src/nvim/tui/tui.c#L1776-L1777

**Comment by dstein64** [comment]
This was addressed in #17197.
